avocado/utils/lv_utils.py

```diff
@@ -54,6 +56,38 @@ def get_diskspace(disk):
     raise LVException('Error in finding disk space')
 
 
+def get_device_total_space(device):
+    """Get the total device size.
+
+    :param str device: name of the device/disk to find the total size
+    :returns: size in bytes
+    :rtype: str
+    :raises: :py:class:`LVException` on failure to find disk space
+    """
+    try:
+        with open('/sys/class/block/{}/size'.format(device), 'r') as fp:
+            blocks = int(fp.read())
+            return blocks * 512
+    except FileNotFoundError:
+        raise LVException('Device not found.')
+
+
+def get_devices_total_space(devices):
+    """Get the total size of given device(s)/disk(s).
+
+    :param list devices: list with the names of devices separated with space.
+    :returns: sizes in bytes
+    :rtype: generator
+    :raises: :py:class:`LVException` on failure to find disk space
+    """
+    size = 0
+    for device in devices.split(" "):
```

Comment:
Maybe you missed my comment here https://github.com/avocado-framework/avocado/pull/3792#discussion_r422107637. You don't need a split, assume that this is a list already.

Reply:
@beraldoleal  hope this is clear now or you are still has concern on splinting devices here.

Reply:
Hi @@Naresh-ibm:

No, it is not clear, yet, so yes, I do have some concerns.

We are in Python, this is a python library and whenever possible, we try to use `lists` instead of `strings` when referring to "list of things". I don't see a reason to use `"sda sdb sdc"` instead of `['sda', 'sdb', 'sdc']`.

We are creating a new method here and it is the perfect opportunity to organize the things here.

We have to design this library thinking on multiple users, not only one specific use case. A different user could think that is better to have a "list of things" as a string but comma-separated: `"sda,sdb,sdc"`. How to make all users happy?

In doubt, we try to use the natural way of the language to express a list, whenever possible.

If the user A, have the data as a string, it is super easy to call this as a list:

```python
total = get_devices_total_space(devices.split())
```

Why are you forcing a list of things to be a string?